src/Lachain.Crypto/DefaultCrypto.cs

```diff
@@ -67,7 +69,7 @@ public bool VerifySignatureHashed(byte[] messageHash, byte[] signature, byte[] p
                     throw new Exception("Cannot serialize parsed key: how did it happen?");
 
                 var parsedSig = new byte[65];
-                var recId = (signature[64] - 36) / 2 / TransactionUtils.ChainId;
+                var recId = (signature[64] - 36) / 2 / TransactionUtils.ChainId (useNewChainId);
```

Comment:
This should be changed because now the recover id could be in two bytes.

Reply:
resolved